.luacheckrc

```diff
@@ -27,6 +27,7 @@ include_files = {
 exclude_files = {
     "lib/tarantool-python",
     "test/test-tarantool/*.test.lua",
+    "test/unittest/*.test.lua",
```

Comment:
Why?

Reply:
Those are `core = tarantool` tests (an input for a console). They poorly compatible with luacheck.

Reply:
Anyway, I would fix luacheck warning with global in config than excluding a whole dir.

```
--- a/.luacheckrc
+++ b/.luacheckrc
@@ -1,4 +1,4 @@
-globals = {"box", "_TARANTOOL", "tonumber64", "os"}
+globals = {"box", "_TARANTOOL", "tonumber64", "os", "test_run"}
 ignore = {
     -- Accessing an undefined field of a global variable <debug>.
     "143/debug",
@@ -27,7 +27,6 @@ include_files = {
 exclude_files = {
     "lib/tarantool-python",
     "test/test-tarantool/*.test.lua",
-    "test/unittest/*.test.lua",
     "lib/luatest/**",
     "lib/checks/**",
 }
```

Reply:
Ok, thanks! Fixed.